changelog.json

```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "1.7.0-beta6": [
+      "[Fixed] Stash viewer does not disable restore button when changes present - #7409",
+      "[Fixed] Stash viewer does not center \"no content\" text - #7299",
+      "[Fixed] Stash viewer pane width not remembered between sessions - #7416",
+      "[Fixed] \"Esc\" key does not close Repository or Branch list - #7177. Thanks @roottool!",
+      "[Fixed] Stash not cleaned up when it conflicts with working directory contents - #7383",
+      "[Improved] Branch names not updated in dialog when stashing and switching branches - #7402",
+      "[Improved] Moved \"Discard all changes\" to Branch menu - #7394. Thanks @ahuth!",
```

Comment:
```suggestion
      "[Improved] Moved \"Discard all changes\" to Branch menu to prevent unintentionally discarding all changes - #7394. Thanks @ahuth!",
```